### PR TITLE
Update ALL the dependencies

### PR DESCRIPTION
**Pull request by SaraVieira**
**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [x] refactoring / style
- [ ] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [x] Yes, I've updated the documentation
- [ ] Not relevant

**Summary**

This PR updates all the dependencies to the last version , fixes the problems that came with it
Also adds prettier to the repo
Removed stage-0 because it's not safe and added babel stage-1 and everything worked fine
Adds instructions for yarn

Update ALL the dependencies

### CODE REVIEW

**Comment by SaraVieira** [comment]
closes #435

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
   "extends": [
-    "okonet"
+    "okonet",
```
Comment (okonet) [inline .eslintrc]: My preset already has prettier so no need in this I guess. I might need to update it to the latest version, though.

File: README.md [hunk 2]
```diff
@@ -23,18 +23,22 @@ Install it from npm and include it in your React build process (using [Webpack](
 ```bash
 npm install --save react-dropzone
 ```
+or:
+```bash
+yarn add react-dropzone --save
```
Comment (okonet) [inline README.md]: AFAIK `yarn add` does `--save` by default

File: package.json [hunk 3]
```diff
@@ -58,45 +58,48 @@
   },
   "dependencies": {
     "attr-accept": "^1.0.3",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-stage-1": "^6.24.1",
+    "es6bindall": "^0.0.9",
```
Comment (okonet) [inline package.json]: Why is this needed?

Comment (SaraVieira) [inline package.json]: Because I forgot to delete it 😢

File: README.md (hunk 2 above)
Comment (SaraVieira) [inline README.md]: Insteresting , didn't know !
My life is a lie :p

File: .eslintrc (hunk 1 above)
Comment (SaraVieira) [inline .eslintrc]: Does it have the plugin and the extend ?

**Comment by SaraVieira** [comment]
@okonet All fixed 😄

**Comment by SaraVieira** [comment]
This can also close all the @greenkeeperio-bot pr's :D

**Comment by okonet** [comment]
Thanks!
